ueber-uns.html

```diff
@@ -0,0 +1,158 @@
+        </div>
+      </div>
+      <div class="image col-md-6 d-flex flex-column justify-content-end">
+        {% picture about-me mission.png --alt Mission %}
+      </div>
+    </div>
+  </div>
+</section>
+<section class="values container">
+  <div class="card overflow-hidden">
+    <div class="row g-0">
+      <div class="image col-md-6 d-flex flex-column justify-content-end">
+        {% picture about-me values.png --alt unsere Werte %}
+      </div>
+      <div class="col-md-6">
+        <div class="card-body p-4 px-lg-5">
+          <h2>Unsere Werte</h2>
+          <ol>
+            <li>
+              <strong>Kompetenz:</strong> Wir streben nach höchster Qualität und
+              Leistung in allem, was wir tun.
+            </li>
+            <li>
+              <strong>Agilität:</strong> Wir passen uns den individuellen
+              Bedürfnissen unserer Teilnehmer und Kunden an.
+            </li>
+            <li>
+              <strong>Innovation:</strong> Wir bleiben stets auf dem neuesten
+              Stand der Technologie und entwickeln kreative Schulungslösungen.
+            </li>
+            <li>
+              <strong>Teamarbeit:</strong> Wir fördern die Fähigkeit, effektiv
+              in multidisziplinären Teams zu arbeiten, um gemeinsame Ziele zu
+              erreichen.
+            </li>
+            <li>
+              <strong>Nachhaltigkeit:</strong> Wir setzen uns für langfristige
+              Erfolge ein.
+            </li>
+          </ol>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+<section class="benefits arguments">
+  <div class="teamtext container">
+    <h2>Unser Team</h2>
+  </div>
+  <div class="team container">
+    <div class="card overflow-hidden">
+      <div class="row g-0">
+        <div class="col-md-6">
+          <div class="card-body p-4 px-lg-5">
+            <p class="founders-heading">Das Gründerpaar</p>
+            <h3 class="card-title h5 mb-1">
+              Clemens Helm
+              <small class="text-muted">Ideengeber und Master Mind der DevCraft Academy. Operativer CEO
+                und CTO. Über 20 Jahre Erfahrung als internationaler Software
+                Entwickler.</small>
+            </h3>
+            <h3 class="card-title h5 mb-1 mt-4">
+              Carola Helm
+              <small class="text-muted">Marketingchefin und strategischer CEO von der ersten Stunde.
+                Koordination der Softskill Trainings.</small>
+            </h3>
+          </div>
+        </div>
+        <div class="image col-md-6 d-flex flex-column justify-content-end founders">
+          {% picture about-me carola_clemens.jpg --alt Clemens und Carola Helm %}
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="teamgallery container">
+    {% for member in site.data.teammembers %}
+    <div class="employee col-md-4">
+      <div class="card">
+        <img src="{{ member.image }}" alt="{{ member.name }}" class="card-img-top" />

```

Comment:
Was mir hier noch auffällt: Verwende bitte auch hier `{% picture %}` statt `img`, damit wir responsive und optimierte Bilder verwenden. Lies dir am besten mal die Docs auf https://rbuchberger.github.io/jekyll_picture_tag/ durch, in unserem Code findest du auch einige Beispiele, wie du es verwenden kannst.